snapshots/run-web.sh

```diff
@@ -1,6 +1,6 @@
 ./node_modules/.bin/selenium-standalone install &&
-./node_modules/.bin/selenium-standalone start & 
-./node_modules/.bin/react-native start --config $(pwd)/rn-cli.config.js --reset-cache &
-./node_modules/.bin/fructose-web -d fructose &
+./node_modules/.bin/selenium-standalone start >> seleniumlog.txt & 
```

Comment:
selenium prints to stderr to ` 2>> ` can also call the file `selenium.log`
can also redirect webpack
./node_modules/.bin/fructose-web -d $(pwd)/fructose  >> webpack.log &